examples/apps/Django_Panel_Multi/Pipfile.lock

```diff
@@ -0,0 +1,535 @@
+{
```

Comment:
I'd prefer not to commit `Pipfile.lock` files, none of the core developers use it. A requirements.txt and/or environment.yml is fine.

Reply:
Ok, I can clean that next week